config/app.json

```diff
@@ -1,3 +1,9 @@
 {
-  "version": "1.123.0"
-}
\ No newline at end of file
+  "version": "1.123.0",
+  "alerts": [
+    {
+      "type": "warning",
+      "message": "This is a warning message"
+    }
```

Comment:
This should be an empty array for now, otherwise it will display after merging

Reply:
Yep, removed!